README.md

```diff
@@ -22,15 +22,24 @@ Similar to the [Rust community's](https://this-week-in-rust.org/), we want to ha
 - Tell active contributors when new versions or tools are available.
 - Attribute developers who have contributed to open-source software related to IPFS.
 
+## Contribute
+
+The Weekly takes a _lot of work_. This work can best be done by being distributed. If you know of anything cool that happens in a given week, anything that other people might be excited about, add it to the weekly by adding a comment about it to the PR or Issue for that week's weekly.
+
+Here are the steps to do that:
+ - Do something awesome, or see something cool.
```

Comment:
I'd swap order here, emphasise that its totally cool to just report something someone else did to the weekly